Java/src/XPlaneConnect.java

```diff
@@ -173,32 +185,27 @@ public XPlaneConnect(String xpHost, int xpPort, int port, int timeout)
     /**
      * Closes the underlying socket.
      */
-    public void close()
-    {
-        if(socket != null)
-        {
+    public void close() {
+        if (socket != null) {
             socket.close();
             socket = null;
         }
     }
 
     /**
-     * Read data from the X-Plane plugin. This method will read whatever data is available and return it.
+     * Read data from the X-Plane plugin. This method will read whatever data is
+     * available and return it.
      *
      * @return The data send from X-Plane.
      * @throws IOException If the read operation fails
      */
-    private byte[] readUDP() throws IOException
-    {
-        byte[] buffer = new byte[65536];
-        DatagramPacket packet = new DatagramPacket(buffer, buffer.length);
-        try
-        {
+    private byte[] readUDP() throws IOException {
+        final byte[] buffer = new byte[65536];
+        final DatagramPacket packet = new DatagramPacket(buffer, buffer.length);
+        try {
             socket.receive(packet);
             return Arrays.copyOf(buffer, packet.getLength());
-        }
-        catch (SocketTimeoutException ex)
-        {
+        } catch (final SocketTimeoutException ex) {
```

Comment:
Most of these formatting changes I'm okay with, but I absolutely cannot stand `catch` (or `else`/`else if`) being on the same line as the previous closing brace

Reply:
Fair enough. These were done automatically by Black, following PEP rules. But I'd rather have this fixed soon than getting stuck in cosmetic things. 😀